### PR TITLE
First draft of Redash working with terraform

### PR DESCRIPTION
**Pull request by pdmthorsrud**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -45,3 +45,9 @@ clean: destroy-box remove-tmp
 update-box:
 	@SSL_CERT_FILE=${SSL_CERT_FILE} CURL_CA_BUNDLE=${CURL_CA_BUNDLE} vagrant box update || (echo '\n\nIf you get an SSL error you might be behind a transparent proxy. \nMore info https://github.com/fredrikhgrelland/vagrant-hashistack/blob/master/README.md#if-you-are-behind-a-transparent-proxy\n\n' && exit 2)
 
+
+
+
+
+
+
```
Comment (fredrikhgrelland) [inline Makefile:53]: remove

File: Makefile [hunk 2]
```diff
@@ -15,7 +15,7 @@ ifdef CUSTOM_CA
 	cp -f ${CUSTOM_CA} docker/conf/certificates/
 endif
 
-up: update-box custom_ca
+up: clean update-box custom_ca
```
Comment (fredrikhgrelland) [inline Makefile:18]: Needed? If so, lets upstream this change.

File: Vagrantfile [hunk 3]
```diff
@@ -3,10 +3,11 @@ Vagrant.configure("2") do |config|
     config.vm.box_version = ">= 0.3, < 0.4"
     config.vm.provider "virtualbox" do |vb|
         vb.linked_clone = true
-        vb.memory = 2048
+        vb.memory = 10000
```
Comment (fredrikhgrelland) [inline Vagrantfile:6]: Do we really need 10000MB memory?
Please consider this amount, and if possible, we should at least keep it under 8GB (8096MB)

File: Vagrantfile [hunk 4]
```diff
@@ -3,10 +3,11 @@ Vagrant.configure("2") do |config|
...
+        vb.cpus = 4
```
Comment (fredrikhgrelland) [inline Vagrantfile:7]: This should be 3 cpus as many machines have 4 cpus, and you will need one to keep vbox and os running.

File: main.tf [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+  detach = false
+  depends_on = [
+    "nomad_job.nomad_job_redash_server",
+  ]
+}
+
+resource "nomad_job" "nomad_job_postgres" {
+  jobspec = data.template_file.nomad_job_postgres.rendered
+  detach = false
+}
```
Comment (fredrikhgrelland) [inline main.tf:55]: Why not use: https://github.com/fredrikhgrelland/terraform-nomad-postgres/releases/tag/0.0.1

Comment (pdmthorsrud) [reply main.tf:55]: This was just a POC to see if it all worked, that's why. We are now adding healthchecks to everything in a new branch. 
Would you like us to use his module in this first PR?

File: Vagrantfile (hunk 3 above)
Comment (pdmthorsrud) [reply Vagrantfile:6]: Not tested at all, was set high just to avoid any problems while developing.

File: Makefile (hunk 2 above)
Comment (pdmthorsrud) [reply Makefile:18]: Not needed, was set while developing, forgot to remove

File: Vagrantfile (hunk 4 above)
Comment (pdmthorsrud) [reply Vagrantfile:7]: Good point

File: main.tf (hunk 5 above)
Comment (fredrikhgrelland) [reply main.tf:55]: nope. Just pointing it out.

**Comment by pdmthorsrud** [comment]
Closing this PR in favor of #3, which has healthchecks added

**Pull request closed without merging**